sci-misc/boinc/boinc-7.6.33.ebuild

```diff
@@ -2,11 +2,11 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
 WX_GTK_VER=3.0
 
-inherit autotools flag-o-matic linux-info systemd user versionator wxwidgets
+inherit autotools linux-info systemd user versionator wxwidgets
```

Comment:
hmmm... right... versionator inherits it. I guess that's why there never was an error.